src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -693,6 +695,10 @@ private boolean shouldTriggerOnEventType(GerritTriggeredEvent event) {
                 return !((WipStateChanged)event).getChange().isWip();
             }
 
+            if(event instanceof HashtagsChanged){
```

Comment:
Why do you need this here? Why isn't implementing this in `shouldTriggerOn` enough?

Reply:
And if it is really needed why are you then iterating over all the configured event types again to find the configuration when you could just either call it or add it as a parameter?

Reply:
I think it could implement in shouldTriggerOn, I will update it.